web/src/lib/components/album-page/thumbnail-selection.svelte

```diff
@@ -10,7 +10,10 @@
   export let album: AlbumResponseDto;
 
   let selectedThumbnail: AssetResponseDto | undefined;
-  const dispatch = createEventDispatcher();
+  const dispatch = createEventDispatcher<{
+    close: void;
+    thumbnailSelected: { asset: AssetResponseDto | undefined };
```

Comment:
Can this just be `thumbnail` and can the data just be AssetResponseDto | undefined?